esp-hal-common/Cargo.toml

```
@@ -54,6 +54,7 @@ esp32   = { version = "0.23.0", features = ["critical-section"], optional = true
 esp32c2 = { version = "0.10.0", features = ["critical-section"], optional = true }
 esp32c3 = { version = "0.13.0", features = ["critical-section"], optional = true }
 esp32c6 = { version = "0.3.0",  features = ["critical-section"], optional = true }
+esp32h2 = { git = "https://github.com/esp-rs/esp-pacs", rev = "3db0042", package = "esp32h2", features = ["critical-section"], optional = true }
```

```suggestion
esp32h2 = { git = "https://github.com/esp-rs/esp-pacs", rev = "0ee1cea", package = "esp32h2", features = ["critical-section"], optional = true }
```

This new commit revision should fix the `rtc_cntl.wdtwprotect` register you had to comment out.